roles/pool/src/lib/mining_pool/mod.rs

```diff
@@ -96,6 +105,33 @@ pub struct Configuration {
     pub test_only_listen_adress_plain: String,
 }
 
+impl Configuration {
+    pub fn new(
```

Comment:
I don't see this function used anywhere?

Reply:
This should be used in tests to initialize config from variables rather than file..

Reply:
it seems clippy is not happy about the number of arguments here: https://github.com/stratum-mining/stratum/actions/runs/10457582329/job/28957531606?pr=1097